### PR TITLE
implemented Completing Party

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issues #:* /bcgov/entity#3394 

*Description of changes:*
- get user email from auth call
- fixed HTML code in dialog (bug /bcgov/entity#3747)
- added user email to completing party
- changed "Not entered" to "(Not entered)" so it doesn't look like data
- added emails to Certify component
- improved Certify styling + cleanup
- deleted unused filing.header.email
- added optional email to org person
- added userEmail to tombstone (and store methods)
- added getCurrentUser() to filing template mixin
- updated certify statements per Scott
- updated unit tests
- misc cleanup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/AddPeopleAndRoles/PeopleAndRoles.vue [hunk 1]
```diff
@@ -7,75 +7,93 @@
...
         @addEditPerson="onAddEditOrgPerson($event)"
         @removePersonEvent="onRemovePerson($event)"
         @resetEvent="resetData()"
-        @removeCompletingPartyRole="removeCompletingPartyAssignment()" />
+        @removeCompletingPartyRole="removeCompletingPartyAssignment()"
+      />
     </v-card>
 
-    <v-card flat v-if="orgPersonList.length > 0" :disabled="showOrgPersonForm" >
+    <v-card v-if="orgPersonList.length > 0" flat :disabled="showOrgPersonForm" >
```
Comment (cameron-freshworks) [inline src/components/AddPeopleAndRoles/PeopleAndRoles.vue:90]: I prefer the conditional first as well :) Easier on the brain for whatever reason hehe

File: src/components/DefineCompany/BusinessContactInfo.vue [hunk 2]
```diff
@@ -6,14 +6,14 @@
         </v-flex>
         <v-flex md4>
           <div><label><strong>Email Address</strong></label></div>
-          <div id="lbl-email">{{ !!contact.email ? contact.email : 'Not entered'}}</div>
+          <div id="lbl-email">{{ !!contact.email ? contact.email : '(Not entered)'}}</div>
```
Comment (cameron-freshworks) [inline src/components/DefineCompany/BusinessContactInfo.vue:9]: Might need a space between `(Not entered)' and the closing }}

File: src/components/DefineCompany/BusinessContactInfo.vue [hunk 3]
```diff
@@ -152,11 +152,12 @@ export default class BusinessContactInfo extends Vue {
 </script>
 
 <style lang="scss" scoped>
-  [class^="col"] {
-    padding-top: 0;
-    padding-bottom: 0;
-  }
-  .business-contact-container {
+[class^="col"] {
```
Comment (cameron-freshworks) [inline src/components/DefineCompany/BusinessContactInfo.vue:155]: Sorry, noob question which i've undoubtedly asked before. What exactly does this syntax mean?

File: src/mixins/legal-api-mixin.ts [hunk 4]
```diff
@@ -106,14 +106,22 @@ export default class LegalApiMixin extends Vue {
...
   }
 
+  /**
+   * Fetches current user data.
+   * @returns a promise to return the user data
+   */
+  getCurrentUser (): Promise<any> {
+    const authUrl = sessionStorage.getItem('AUTH_API_URL')
+    const config = { baseURL: authUrl }
+    return axios.get('users/@me', config)
```
Comment (cameron-freshworks) [inline src/mixins/legal-api-mixin.ts:122]: Interesting url!

File: src/components/DefineCompany/BusinessContactInfo.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/DefineCompany/BusinessContactInfo.vue:9]: The linter (and Vue) doesn't care about this but for the sake of consistency, I will update!

File: src/components/DefineCompany/BusinessContactInfo.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/DefineCompany/BusinessContactInfo.vue:155]: This selects every element whose class attribute value begins with "col".

https://www.w3schools.com/cssref/css_selectors.asp

Comment (severinbeauvais) [reply src/components/DefineCompany/BusinessContactInfo.vue:155]: (^ is start of line in vi; $ is end of line)

File: src/components/DefineCompany/BusinessContactInfo.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/DefineCompany/BusinessContactInfo.vue]: Fixed.

**Pull request merged**